src/rng/backends/rocrand/CMakeLists.txt

```diff
@@ -64,7 +64,7 @@ add_library(${LIB_OBJ} OBJECT ${SOURCES})
 
 target_include_directories(
   ${LIB_OBJ} PRIVATE ${PROJECT_SOURCE_DIR}/include ${PROJECT_SOURCE_DIR}/src
-                     ${CMAKE_BINARY_DIR}/bin ${MKL_INCLUDE})
+                     ${CMAKE_BINARY_DIR}/bin ${MKL_INCLUDE} ${ONEMKL_GENERATED_INCLUDE_PATH})
```

Comment:
Could you please check that clang-format won't complain about this line?

Reply:
Checked Clang format using our regular CI job. Everything is fine.